qiskit/circuit/library/__init__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2017, 2019.
```

Comment:
```suggestion
# (C) Copyright IBM 2017, 2019, 2020.
```
or only 2020? And where's 2018? How does one determine the years in the copyright?

Reply:
yes our convention is all over the place. i think just `2020` as the last year the code was updated is sufficient, but i'm not sure. but it needs cleaning all over the code.